site/docs/4.5.1/admin/upgrade.md

```diff
@@ -59,15 +59,6 @@ For each Bookie:
 
 We describes the general upgrade method in Apache BookKeeper as above. We will cover the details for individual versions.
 
-### 4.4.x to 4.5.x upgrade
```

Comment:
can you keep 4.4.x to 4.5.x upgrade there? why do you need to remove it?

Reply:
done